.gitignore

```diff
@@ -1,6 +1,6 @@
 build/
 
-Systemtheorie.pdf
+/*.pdf
```

Comment:
This can cause images to be ignored. We must be explicit here

Reply:
I think the slash causes it to only ignore files in the root directory (excluding subdirectories)

Reply:
And if someone add a PDF into the root dir?

Reply:
Yes, any pdf files directly in the root directory would be ignored. PDF's in other directories such as `img` will not be ignored.

Reply:
My question is if this is the expected behavior or if we should stick to explicitly ignoring this single file.

Reply:
I think it is a good practice to have all image files in a separate directory. So for me this is expected behavior, but I can also change it back to only ignore the single file.

Reply:
```suggestion
*.pdf
!img/*.pdf
```

How about this?

Reply:
Okay